EmbodiedSuffering_oM/Elements/Slavery.cs

```diff
@@ -32,10 +32,19 @@
 namespace BH.oM.EmbodiedSuffering.Elements
 {
     [Description("An Embodied Suffering object used for defining the prevalence of modern slavery in each country's population. Measured by the number of victims per 1000 people.")]
```

Comment:
An Embodied Suffering object used to estimate the risk of labour exploitation by measuring number of victims of modern slavery per 1000 people, and violations against workers' rights. Later updates may include monitoring worker voice and monitoring other socioeconomic factors that lead to exploitation.